src/test/codegen/c-variadic.rs

```diff
@@ -1,3 +1,4 @@
+// ignore-emscripten compiled with panic=abort by default
```

Comment:
The test is about variadics, not panics, so I do not understand this comment.

(I realize it's been a while, but I just coincidentally noticed this comment and traced it back to where it was added...)

Reply:
My guess is that compiling with `panic=abort` changes the codegen in such a way that some of the CHECK lines no longer pass (particularly down in `test_c_variadic_call`, which does not have an explicit panic annotation).